portfolio/src/main/webapp/jquery_functions.js

```diff
@@ -0,0 +1,31 @@
+/*
+* Cycles between  different background images, triggered by mouse click
+*/
+
+let img_arr = new Array(5);
+img_arr = ["../images/mountbierstadt.jpg", 
+           "../images/mountyale.jpg",
+           "../images/gardenofgods.jpg",
+           "../images/lakedillon.jpg",
+           "../images/mountelbert.jpg"
+           ];
+labels = ["the summit of Mount Bierdstadt.", 
+          "the summit of Mount Yale.", 
+          "the Garden of the Gods.", 
+          "Lake Dillon, seen from the vantage point of driving along Dillon Dam road.",
+          "the summit of Mount Elbert, the tallest point in Colorado."];
+
+
+$(document).ready(function(){
+    // if change background button is clicked, background image for the body is changed. 
+    // Appropriate image label is displayed.
+  let index = 0;
```

Comment:
Nit: I think index is a little too generic for what it is being used for. A descriptive variable name might help here, especially since it is one that is keeping track of state.